shotover-proxy/example-configs/redis-cluster-auth/topology.yaml

```diff
@@ -5,6 +5,10 @@ sources:
       listen_addr: "127.0.0.1:6379"
 chain_config:
   redis_chain:
+    # This transform is only here to ensure that the transform correctly handles the case where
+    # redis returns an error due to being unauthenticated
```

Comment:
Can we move this to `test-configs`? It's no different from the other cluster ones and now contains testing specific stuff.

Reply:
ah true, i'll do in a follow up.